tests/behat/features/install/users/RoleContributor.feature

```diff
@@ -85,6 +89,7 @@ Feature: Install State
     And drush output should contain "view hs_person revisions"
     And drush output should contain "view hs_publications revisions"
     And drush output should contain "view hs_research revisions"
+    And drush output should contain "view own field_paragraph_style"
```

Comment:
it's interesting that you can distinguish between "your own" field and "anybody's" field